### PR TITLE
Fix ActiveRecord::PendingMigrationError action not dumping schema.

### LINKED ISSUE #40105: Actionable PendingMigrationError does not dump schema
### Steps to reproduce
1. Create a pending migration
2. Visit localhost:3000
3. See a `PendingMigrationError` error screen with a button to run migrations
4. Click the button

### Expected behavior
Migrations are executed and the schema/structure.sql is dumped

### Actual behavior
Migrations are executed

### System configuration
**Rails version**: 6.0.3.2

**Ruby version**: 2.6.6

### PR DESCRIPTION
**Pull request by tgxworld**

### CODE REVIEW

**Comment by tgxworld** [comment]
@kaspth Do you mind reviewing this for me? I took your suggestion in https://github.com/rails/rails/pull/34788/files#r264059772 and implemented it as part of the fix.

File: activesupport/lib/active_support/actionable_error.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require "rails/command"
```
Comment (rafaelfranca) [inline activesupport/lib/active_support/actionable_error.rb:3]: Active Support can't depend on Railties. We should register the command as the action to run on that exception in a Railtie if we want to use Rails Commands for them.

**Review by rafaelfranca — CHANGES_REQUESTED** [review]

**Comment by tgxworld** [comment]
@rafaelfranca I've updated the PR to not use Rails command as it seems to be dependent on the current path which it is executed from. Also, using Rails command means we lose multi-database support for check pending errors as well.

**Review by rafaelfranca — APPROVED** [review]

**Comment by philipbjorge** [comment]
Thanks @tgxworld -- Looking forward to getting this to our dev team!

**Comment by tgxworld** [comment]
Thank you for reviewing @rafaelfranca :+1:

**Pull request merged**